recipes/pybdei/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pybdei" %}
+{% set version = "0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pybdei-{{ version }}.tar.gz
+  sha256: e42082d4c16aa829e2857cb5b5760f3f12f735d288e302843980f49992907111
+
+build:
+  script_env:
+    - CC=g++
+    - CXX=g++
```

Comment:
```suggestion
```
`$CXX` will be exported by the build and point to `$BUILD_PREFIX/bin/x86_64-conda-linux-gnu-c++`